src/note_item.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTrash, faExpandArrowsAlt, faPencilAlt } from '@fortawesome/free-solid-svg-icons';
+import Draggable from 'react-draggable';
+import marked from 'marked';
+
+
+class NoteItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isEditing: false, text: this.props.note.text, title: this.props.note.title, x: this.props.note.x, y: this.props.note.y,
```

Comment:
Here you have a lot of local state in note. The reason why your changes are not updating across browsers is because isEditing is local and not being sent up